### PR TITLE
Release v0.6.2

### PR DESCRIPTION
**Pull request by shiftkey**
- [x] update release notes
- [x] version bump
- [x] :thumbsup: on release notes
- [x] integration tests all :ok_hand: 
- [x] :ship: to MyGet
- [x] :shipit:

### CODE REVIEW

File: Octokit/Http/InMemoryCredentialStore.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ public InMemoryCredentialStore(Credentials credentials)
 
         public Task<Credentials> GetCredentials()
         {
-            return Task.Factory.StartNew(() => _credentials);
+            return Task.FromResult(_credentials);
```
Comment (shiftkey) [inline Octokit/Http/InMemoryCredentialStore.cs]: This actually introduces some resource contention when parallelizing tests (but isn't the only one).

Do you have any objections to this @haacked?

File: Octokit/Http/JsonHttpPipeline.cs [hunk 2]
```diff
@@ -48,12 +48,15 @@ public void DeserializeResponse<T>(IResponse<T> response)
...
-                if (body != "{}") 
+                if (body != "{}")
                 {
+                    var typeIsDictionary = typeof(IDictionary).IsAssignableFrom(typeof(T));
+                    var typeIsEnumerable = typeof(IEnumerable).IsAssignableFrom(typeof(T));
+                    var responseIsArray = body.StartsWith("{", StringComparison.Ordinal);
+
                     // If we're expecting an array, but we get a single object, just wrap it.
                     // This supports an api that dynamically changes the return type based on the content.
-                    if ((typeof(IEnumerable).IsAssignableFrom(typeof(T)))
```
Comment (shiftkey) [inline Octokit/Http/JsonHttpPipeline.cs]: `IDictionary` subclasses `IEnumerable`, so some of our tests which return `Dictionary<T1,T2>` were being caught here and deserialized incorrectly.

Feel free to submit a PR to make this a bit more cleaner.

File: Octokit/Clients/RepositoriesClient.cs [hunk 3]
```diff
@@ -386,18 +386,17 @@ public Task<IReadOnlyList<RepositoryContributor>> GetAllContributors(string owne
...
             Ensure.ArgumentNotNullOrEmptyString(name, "name");
 
-            return ApiConnection
-                .Get<IDictionary<string, long>>(ApiUrls.RepositoryLanguages(owner, name))
-                .ContinueWith<IReadOnlyList<RepositoryLanguage>>(t => 
-                    new ReadOnlyCollection<RepositoryLanguage>(
-                        t.Result.Select(kvp => new RepositoryLanguage(kvp.Key, kvp.Value)).ToList()
-                    ),
-                    TaskContinuationOptions.OnlyOnRanToCompletion);
+            var data = await ApiConnection
```
Comment (shiftkey) [inline Octokit/Clients/RepositoriesClient.cs]: Just tidied up this function to behave like `GetEmoji` while I was in changing `IDictionary` to `Dictionary`

File: Octokit/Http/InMemoryCredentialStore.cs (hunk 1 above)
Comment (Haacked) [inline Octokit/Http/InMemoryCredentialStore.cs]: No objections. In fact, I meant to make this change everywhere. We should never be using `StartNew` just to wrap an object in a task.

Comment (shiftkey) [inline Octokit/Http/InMemoryCredentialStore.cs]: I'll clean that up against `master` once this is merged in.

Comment (shiftkey) [inline Octokit/Http/InMemoryCredentialStore.cs]: Also, I can see a `Task.Factory.StartNew` inside `HttpClientHandler.SendAsync` that also causes contention (I see four requests enter, but none complete).

**Comment by shiftkey** [comment]
Can I get a :thumbsup: on the release notes? Everything else is ready to roll out after that...

**Comment by Haacked** [comment]
:thumbsup: to release notes

**Comment by shiftkey** [comment]
And we're live: https://www.nuget.org/packages/Octokit/0.6.2

**Pull request merged**